src/sqlfluff/rules/L041.py

```diff
@@ -61,7 +76,12 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             ]
             fixes = [
                 # E.g. "\n" -> " DISTINCT\n.
-                LintFix("edit", newline, replace_newline_with),
```

Comment:
Unrelated fix, but... I have recently found several inappropriate uses of `"edit"` fixes. Edits replace a single existing segment with a new segment of the same type, and the specified segments as children. This is **very often a mistake**, because you're adding child nodes rather than sibling nodes. Changing the structure of the parse tree often violates the assumptions implicit in the dialect (i.e. the segment and grammar definitions), and is prone to causing very weird interactions (i.e. **bugs**) together with other rules.